.github/scripts/trymerge.py

```diff
@@ -967,12 +970,15 @@ def add_conclusions(edges: List[Dict[str, Dict[str, Any]]]) -> None:
             checkruns = node["checkRuns"]
             if workflow_run is not None:
                 conclusions[workflow_run["workflow"]["name"]] = (node["conclusion"], node["url"])
+            check_run_name_prefix = "" if workflow_run is None else f'{workflow_run["workflow"]["name"]} / '
```

Comment:
Can we extract this out to a method? It's repeating some stuff and it's a bit hard to parse.

Reply:
+1 I confuse myself looking at the 2 "almost" identical `add_conclusions` functions in `get_checkrun_conclusions` and `get_land_checkrun_conclusions`. It's good to have them refactored into one. It can be in another PR though because it's out of scope for this one